openupgradelib/openupgrade.py

```diff
@@ -1899,6 +1901,39 @@ def get_model2table(model):
     return model2table.get(model, model.replace(".", "_"))
 
 
+def get_field2column_type(field_type):
```

Comment:
This is still missing `translate=False` keyword argument.

Reply:
It's not missing. It's a parameter of `add_columns`.

Reply:
But semantically it should be here, as this method intents to return the absolute SQL type for any field type.

Reply:
Done.